app/controllers/collections_controller.rb

```diff
@@ -121,6 +121,29 @@ def revoke_access
     redirect_to licences_path
   end
 
+  def add_items_to_collection
+    # referenced documents (HCSVLAB-1019) are already handled by the look_for_documents part of the item ingest
+    begin
+      request_params = cleanse_params(params)
+      collection = validate_collection(request_params)
+      corpus_dir = corpus_dir(request_params[:id])
+      validate_add_items_request(collection, corpus_dir, request_params)
+      uploaded_files = process_uploaded_files(corpus_dir, collection.name, request_params[:file])
+      items = []
```

Comment:
everything from `items = []` up to and including `raise ResponseError.new(400), "No items were added" if items.blank?` would be better off in `process_items` helper method